components/Episodes.vue

```diff
@@ -6,7 +6,7 @@ import VCard from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VCar
 import VFlexibleLink from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VFlexibleLink.vue'
 import PlaySelector from '~/components/PlaySelector.vue'
 
-const props = defineProps({
```

Comment:
What are you using for linting/style-control? These spaces are super non-standard. This isn't a blocker, just a question.

Reply:
Bono and I are discussing it - he wanted to use Volar but it keeps adding these damn spaces